subprojects/provider/src/main/kotlin/org/gradle/kotlin/dsl/DependencyConstraintHandlerScope.kt

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright 2018 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.kotlin.dsl
+
+import org.gradle.api.artifacts.Configuration
+import org.gradle.api.artifacts.DependencyConstraint
+import org.gradle.api.artifacts.dsl.DependencyConstraintHandler
+
+
+/**
+ * Receiver for `dependencies.constraints` block providing convenient utilities for configuring dependency constraints.
+ *
+ * @see [DependencyConstraintHandler]
+ */
+class DependencyConstraintHandlerScope(val constraints: DependencyConstraintHandler) : DependencyConstraintHandler by constraints {
```

Comment:
Shouldn't this be annotated with `@Incubating` the same way the underlying API is?